server/api/scheduler.go

```diff
@@ -113,11 +113,16 @@ func (h *schedulerHandler) CreateScheduler(w http.ResponseWriter, r *http.Reques
 			return
 		}
 	case types.GrantLeaderScheduler, types.EvictLeaderScheduler:
-		storeID, ok := input["store_id"].(float64)
+		_, ok := input["store_id"]
```

Comment:
Why?

Reply:
`_, ok := input["store_id"]` represents the input("store_id") doesn't exist.

`storeID, ok := input["store_id"].(float64)` represents the input("store_id") can not be convert to float.

Reply:
Is it necessary?

Reply:
I think we can put it here. It won't be bad at all.